libs/core/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@kirbydesign/core",
-  "version": "0.0.58",
+  "version": "0.0.59",
```

We are bundling many features into coming releases, so a bump is not necessary in specific PRs :) 

```suggestion
  "version": "0.0.58",
```